index.html

```diff
@@ -615,6 +622,10 @@ <h2>Creating a navigation timing entry</h2>
         <li>Set |document|'s <span>navigation timing entry</span> to |navigationTimingEntry|.
         <li>Set |navigationTimingEntry|'s <a data-for="PerformanceNavigationTiming">`Critical-CH` restart time</a>
         to |criticalCHRestart|.
+        <li>If |document|'s [=Document/not restored reasons=] is not null, then set
```

Comment:
document's or document's state's? I guess those are the same thing, but oddly enough the HTML spec pr talks about them both.

Reply:
Hmm, why do we need the null check here.  Not restored reasons should be initialized with some value. Either with null or valid reasons object.

Reply:
You're right, I removed the null check.